playwright/tests/general_regression/sequencing.spec.js

```diff
@@ -9,27 +9,36 @@ test('Add, edit and remove a Sequence', async ({ page }) => {
 
   // Go to https://staging.talentticker.ai/en-GB/home
   await page.goto('https://staging.talentticker.ai/en-GB/home');
-  // Click [data-test="contatcsNavButton"] svg
-  await page.click('[data-test="contatcsNavButton"] svg');
+
+  // Click [data-test="contatcsNavButton"]
+  await page.click('[data-test="contatcsNavButton"]');
```

Comment:
I am guessing this spelling mistake is a Dev one?

Reply:
Yep, seemed like a lot of work to change it so kept it for now.